cmd/security-proxy-setup/res/configuration.toml

```diff
@@ -93,4 +95,10 @@ RetryWaitPeriod = "1s"
   Name = "virtualdevice"
   Protocol = "http"
   Host = "localhost"
-  Port = 49990
\ No newline at end of file
+  Port = 49990
+
+  [Routes.edgex-core-consul]
+  Name = "consul"
+  Protocol = "http"
+  Host = "localhost"
```

Comment:
Is the env var override for Docker already taken care of?

Reply:
yea, this will be done in edgex-compose

Reply:
added in this PR: https://github.com/edgexfoundry/edgex-compose/pull/73/files